e2e/config.js

```diff
@@ -32,11 +34,11 @@ module.exports = {
       },
       {
         expectedPrompt: /What is the URL for your Commerce Cloud instance?/i,
-        response: "https://zzte-053.dx.commercecloud.salesforce.com\n",
+        response: "https://zzrf-002.dx.commercecloud.salesforce.com\n",
```

Comment:
Why are we using zzrf-002 rather than zzte-053 or zzrf-001?

Reply:
`zzte-053` is owned by "Salesforce B2C Temporary external users" org and we'd like to have something with greater control/ownership should we need to make changes. Umm no specific reason for picking `zzrf-002` over `zzrf-001` more like a clean setup.